src/foam/nanos/approval/ApprovableAwareDAO.js

```diff
@@ -382,28 +382,14 @@ foam.CLASS({
           return obj;
         }
 
-        // change last modified by to be the user
-        updatedProperties.put("lastModifiedBy", user.getId());
-
-        // remove lastmodified prop to exclude it from hash
-        Object lastModified = updatedProperties.remove("lastModified");
-
+        String approvableHashKey = ApprovableAware.getApprovableHashKey(x, obj);
         DAO approvableDAO = (DAO) x.get("approvableDAO");
 
-        String daoKey = "d" + getDaoKey();
```

Comment:
this is going to break the approval unit tests which rely on this key structure.

Reply:
what's the name of the tests relying on this? I ran tests just now but didnt see anything break

Reply:
./build.sh -cjQT LiquidApprovalTest

Reply:
it's this query .. the CONTAINS_IC:
```
    approvalRequests = (ArraySink) this.getApprovalRequestDAO(x).inX(this.getSystemX()).where(AND(
      CONTAINS_IC(ApprovalRequest.OBJ_ID, String.valueOf(user.getId())),
      EQ(ApprovalRequest.CLASSIFICATION, "User"),
      EQ(ApprovalRequest.OPERATION, operation),
      EQ(ApprovalRequest.IS_FULFILLED, false)
    )).select(new ArraySink());```

Reply:
added back a simplified hashedid to account for this